infra/build/functions/cloud_build_emulator.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
```

Comment:
Should we look at using https://github.com/GoogleCloudPlatform/cloud-build-local instead? It's unmaintained, but it should at least be closer to real cloud build than anything we write. 

There's probably specifics here that we're missing (e.g. the "cloudbuild" network) which might subtly break things otherwise.

Reply:
>Should we look at using https://github.com/GoogleCloudPlatform/cloud-build-local instead? It's unmaintained, but it should at least be closer to real cloud build than anything we write.

It sucks and completely failed on everything I tried it on. But I could be holding it wrong, maybe someone else should try it.

Reply:
I agree with you that it's a better solution in theory. This was a last resort after realizing there was no other option that could give us builds with VM level isolation.

Reply:
What did we need the VM level isolation for btw? Is this for OSS-Fuzz-Gen?

Reply:
For OSS-Fuzz as well.